src/components/Beerdetails.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import {useState, useEffect} from 'react'
+import {Link} from 'react-router-dom'
+import axios from 'axios'
+import Navbar from './Navbar';
+import {Routes, Route, useParams} from 'react-router-dom'
+
+
+
+function Beerdetails() {
+   const {beerId} = useParams();
+   const [oneBeer, setOneBeer] = useState(null);
```

Comment:
here, the useState should be set to {} instead of null

`const [oneBeer, setOneBeer] = useState({});`